tasks/pipeline.py

```diff
@@ -810,6 +810,24 @@ def update_circleci_config(file_path, image_tag, test_version):
         circle.write(circle_ci.replace(f"{match.group(0)}", f"{image}:{image_tag}\n"))
 
 
+@task(
+    help={
+        "file_path": "path of the CircleCI configuration YAML file",
+    }
+)
```

Comment:
```suggestion
@task(
    help={
        "file_path": "path of the CircleCI configuration YAML file",
    },
    autoprint=True
)
```

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Same as above